README.md

```diff
@@ -75,6 +75,11 @@ the binary verification command above.
 
 ## Build
 
+To quickly test that io.js was built correctly from any of the steps below:
```

Comment:
I'm with @cjihrig. Why test something that isn't built yet? Regarding people not reading a readme -- well, that's not too much to do about.